### DISCUSSION TITLE
Collection of Research, Articles, Data

### DISCUSSION THREAD

**Discussion by rjheby** [PullDating/Org#122] [General]
Growth Marketing: 
[Boosting the Audience from 12M to 375M: the Unique Story of Badoo](https://medium.com/@humanoit/boosting-the-audience-from-12m-to-375m-myth-or-reality-dbfc38e9540b)
[How to Gather 100,000 Emails in One Week (Includes Successful Templates, Code, Everything You Need)](https://tim.blog/2014/07/21/harrys-prelaunchr-email/)

Research & Data
[OkCupid: Exactly What To Say In A First Message](https://theblog.okcupid.com/exactly-what-to-say-in-a-first-message-2bf680806c72)
[Nearly Half of U.S. Adults Say Dating Has Gotten Harder for Most People in the Last 10 Years](https://www.pewresearch.org/social-trends/2020/08/20/nearly-half-of-u-s-adults-say-dating-has-gotten-harder-for-most-people-in-the-last-10-years/)

Reddit
[Actual men on tinder](https://www.reddit.com/r/Tinder/comments/v1yg8o/comment/iaro67j/?utm_source=reddit&utm_medium=web2x&context=3)
[A Pew Research Center study shows that a majority of single women in the U.S. are no longer on the dating market, as in not looking for marriage, a committed relationship or casual sex. What do you think explains this drop?](https://www.reddit.com/r/dating/comments/v0j7sr/a_pew_research_center_study_shows_that_a_majority/)

**Comment by rjheby** [comment]
Reciprocity as a business model https://briefthoughts.substack.com/p/reciprocity-as-a-business-model?s=r&utm_campaign=post&utm_medium=web&utm_source=direct

**Comment by Oshyan** [comment]
Research is also to be found in the Wiki: https://github.com/PullDating/Org/wiki/Existing-Articles,-Discussion,-Communities,-etc.
